### ISSUE TITLE
React component triggers aria-role

### ISSUE THREAD

**Issue by merlinpatt** [evcohen/eslint-plugin-jsx-a11y#198]
I have a React component that is triggering the aria-role rule. This rule should not trigger on React components.

This React component

`<ChooseRoleButton onLoginPage={onLoginPage} role="teacher" />`

gives me this error

`Elements with ARIA roles must use a valid, non-abstract ARIA role  jsx-a11y/aria-role`

**Comment by evcohen** [comment]
Ah, this is fixed but undocumented (#177). Must pass in option to `ignoreNonDom: true` to the rule in the config to resolve this issue

**Comment by merlinpatt** [comment]
So the rule should look like this?

```
'jsx-a11y/aria-role': ['error', {ignoreNonDom: true}],
```

**Comment by ljharb** [comment]
DOM is capitalized, since it's an initialism. Docs merged in #199.
